src/bridge/Makefile.am

```diff
@@ -39,6 +39,7 @@ libcockpit_metrics_a_SOURCES = \
 	src/bridge/cockpitsamples.h \
 	$(NULL)
 
+# TODO: remove all of the below
 if WITH_OLD_BRIDGE
```

Comment:
FTR: I tried with `if 0` or `if REMOVE_ME` etc., but none of that works. There just isn't a multi-line "disable this section" in automake that I know of :cry: I am happy to just remove the block here, but then we should also remove all the files (as this Makefile block is the reference list of which files  can go), and then I step too much on Lis'es toes.